graphql-dgs-client/src/main/kotlin/com/netflix/graphql/dgs/client/GraphQLClient.kt

```diff
@@ -67,7 +67,7 @@ interface GraphQLClient {
 
     companion object {
         @JvmStatic
-        fun createCustom(url: String, requestExecutor: RequestExecutor) = CustomGraphQLClient(url, requestExecutor)
+        fun createCustom(url: String, requestExecutor: RequestExecutor, customDeserializer: Map<Class<*>, (Any) -> Any?>? = null) = CustomGraphQLClient(url, requestExecutor, customDeserializer)
```

Comment:
same as with the constructor, you can't just add arguments. It would work for Kotlin, but not for Java